package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pinwheel/react-modal",
-  "version": "2.3.0",
+  "version": "2.4.0",
```

Comment:
[q] Link is on 2.3, shouldn't this be capped at 2.3?

Reply:
Good catch! brain is off 🤦